core/src/main/scala/sttp/example/Main.scala

```diff
@@ -0,0 +1,10 @@
+package sttp.example
+
+import org.openapitools.client.api.PetApi
+import org.openapitools.client.model.Pet
+
+object Main {
+  def main(args: Array[String]): Unit = {
+    PetApi()(???).addPet(Pet(Some(123), photoUrls = Seq.empty, name = "reks"))
```

Comment:
maybe there's an on-line running version of petstore which we could query here? :)